src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -139,6 +139,8 @@
   </ItemGroup>
   <PropertyGroup Condition="$(RequiresAlmAuthentication)=='true'">
     <DisableStrongNamer Condition="'$(SignAssembly)' != 'true'">true</DisableStrongNamer>
+    <!-- Tell the StrongNamer to use our key, if set. -->
+    <StrongNamerKeyFile>$(AssemblyOriginatorKeyFile)</StrongNamerKeyFile>
```

Comment:
I made this change in the first commit, expecting the CI build to fail with a "assembly refs have changed error", [which it did](https://sonarsource.visualstudio.com/DotNetTeam%20Project/_build/results?buildId=18669&view=logs&j=71179031-6417-5a2f-3d87-af6fce2011e4&t=ecf70746-8ac8-5b27-82e5-c7e3acc6f797&l=3210). Hurrah!

```
         ***** D:\A\1\S\SRC\INTEGRATION.VSIX\AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt
         - 'EnvDTE80, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
         - 'Microsoft.Alm.Authentication, Version=4.0.0.0, Culture=neutral, PublicKeyToken=e33b67d3bb5581e4'
         - 'Microsoft.VisualStudio.CodeAnalysis, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
         ***** D:\A\1\S\SRC\INTEGRATION.VSIX\BIN\RELEASE\VS2019\NET46\ASMREF_INTEGRATION.VSIX_LATEST.TXT
         - 'EnvDTE80, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
         - 'Microsoft.Alm.Authentication, Version=4.0.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
         - 'Microsoft.VisualStudio.CodeAnalysis, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
         *****
```

Reply:
Awesome! 😸